### ISSUE TITLE
Improvements for ORT Run details page

### ISSUE THREAD

**Issue by Etsija** [eclipse-apoapsis/ort-server#293]
From 23.5.2024 front-end alignment meeting:

- [x]  move downloadable results to the top of page (before JSONs)
- [ ] download logs from a run - currently blocked by issue #168 
- [ ] implement streaming logs instead of / in addition to downloadable ones
- [x] thinking forward, the JSONs are there mainly for debugging purposes currently, so a nicer way of showing job configs and results is needed

**Comment by mnonnenmacher** [comment]
>     * thinking forward, the JSONs are there mainly for debugging purposes currently, so a nicer way of showing job configs and results is needed

I think the UI for that should look similar to the one used when starting a new run.

**Comment by sschuberth** [comment]
The item

> * implement streaming logs instead of / in addition to downloadable ones

is neither ticked nor tracked as a separate issue... @Etsija please take care that it does not fall through the cracks.
